source/index.html.md

```diff
@@ -251,29 +248,49 @@ This evente is triggerend once the user unfavorites or unpins a favorited listin
 
 ```json
 //DATA EXAMPLE
-{
-  "search": {
-    "id": "search_1",
-    "bathroom_count": 1,
-    "bedroom_count": 2,
-    "location_name": "boston",
-    "location_level": "place", // state|county|metro|place|nbh
-    "max_price_cents": 100000000,
-    "min_price_cents": "",
-    "listing_image_cover_url": "https://listing-images.nestready.net/dArboFrG37bMhDjHxWUMBZ3vs",
-    "property_type": "apartment/condo", // townhouse|single-family-home|all
-  }
-}
+{"searches":
+    [{"id":"5d71461ad71b0100091a10af",
+      "bathrooms_count":"3",
+      "bedrooms_count":"3",
+      "max_price_cents":"400000",
+      "min_price_cents":"1",
+      "listing_image_cover_url":"",
+      "property_type":"townhouse",
+      "commute":{"mode":"car", "range":"20", "name":"work"},
+      "location":{"id":"597796fbb5e1b5f5d7000655", "name":"boston metro area", "level":"metro"}},
+     {"id":"5d714622d71b0100091a10b0",
+      "bathrooms_count":"3",
+      "bedrooms_count":"3",
+      "max_price_cents":"400000",
+      "min_price_cents":"1",
+      "listing_image_cover_url":"",
+      "property_type":"townhouse",
+      "commute":{"mode":"car", "range":"20", "name":"work"},
+      "location":{"id":"597796fbb5e1b5f5d7000655", "name":"boston metro area", "level":"metro"}},
+     {"id":"5d714799d71b0100091a10b3",
+      "bathrooms_count":"3",
+      "bedrooms_count":"3",
+      "max_price_cents":"400000",
+      "min_price_cents":"1",
+      "listing_image_cover_url":"",
+      "property_type":"townhouse",
+      "commute":{"mode":"car", "range":"20", "name":"work"},
+      "location":{"id":"597796fbb5e1b5f5d7000655", "name":"boston metro area", "level":"metro"
+      }
+    }
+    ]
+  },
+ "event_type":"saved_search_updated"}
 ```
 This event is triggered once the user, using the filter provided, performs a property search
 
-### search_saved
+### saved_search_updated
```

Comment:
@natashareis  @AlexVKO Does this event already exist or it's going to be implemented soon?

Reply:
Resolving as @natashareis said it's being implemented.